### DISCUSSION TITLE
TJC screen cannot be updated (change icon asset)

### DISCUSSION THREAD

**Discussion by OniLink57** [mriscoc/Ender3V2S1#689] [Q&A]
Hi,

Thanks for your firmware. I have a gen 4 screen (https://marlin.crc.id.au/forum/t/identifying-ender-3-v2-screen-versions/960) and with the "stock" Marlin firmware, I couldn't properly use it because I can't update the screen with the proposed firmware.
I can't update the screen with your firmware neither, but at least it works. I just don't have the mesurements when I calibrate the plate.

So that's more than an information than a bug report. We currently can't update gen 4 screens with the previous methods. It just doesn't work without the TJC_SET directory.

**Comment by mriscoc** [comment]
You have a TJC display https://github.com/mriscoc/Ender3V2S1/wiki/How-to-update-the-display

**Reply by OniLink57** [reply]
Exactly :) And right now I can't update it so I'm glad this firmware works without it !
